packages/react/src/experimental/Widgets/Content/ListItems/WidgetAvatarsListItem/index.tsx

```diff
@@ -0,0 +1,59 @@
+import { cn } from "../../../../../lib/utils"
+import { AvatarVariant } from "../../../../Information/Avatars/Avatar"
+import { AvatarList } from "../../../../Information/Avatars/AvatarList/index.tsx"
+import { EmojiAvatar } from "../../../../Information/Avatars/EmojiAvatar"
+
+export type Props<Id extends string | number = string | number> = {
+  id: string | number
+  emoji: string
+  title: string
+  subtitle: string
+  avatars: AvatarVariant[]
+  onClick?: (id: Id) => void
+}
+
+type WrapperProps = {
+  onClick?: (ev: React.MouseEvent<HTMLAnchorElement>) => void
+  children: React.ReactNode
+}
+
+const Wrapper: React.FC<WrapperProps> = ({ onClick, children }) => {
+  const className = cn(
+    "flex flex-row gap-2 rounded-md border border-solid border-transparent p-2 text-f1-foreground",
+    onClick
+      ? "cursor-pointer hover:bg-f1-background-tertiary focus:border-f1-background-selected-bold focus:outline-none"
+      : undefined
+  )
+  return onClick ? (
+    <a className={className} onClick={onClick} tabIndex={0}>
```

Comment:
It is selectable by default
```suggestion
    <a className={className} onClick={onClick}>
```